lib/form/SettingsOaiRepositoryForm.class.php

```diff
@@ -41,7 +41,7 @@ public function configure()
       'oai_repository_identifier' => new sfWidgetFormInput(array(), array('class'=>'disabled', 'disabled'=>true)),
       'sample_oai_identifier' => new sfWidgetFormInput(array(), array('class'=>'disabled', 'disabled'=>true)),
       'resumption_token_limit' => new sfWidgetFormInput,
-      'oai_additional_sets_enabled' => new sfWidgetFormSelectRadio(array('choices'=>array(1=>'yes', 0=>'no')), array('class'=>'radio'))
+      'oai_additional_sets_enabled' => new arWidgetFormSelectRadioOaiAdditionalSetEnable(array('choices'=>array(1=>'yes', 0=>'no')), array('class'=>'radio'))
```

Comment:
We usually add spaces between the array arrows. It'd be nice if you could fix that in this widgets block.